_delphi_utils_python/delphi_utils/weekday.py

```diff
@@ -40,7 +41,15 @@ def get_params(data, denominator_col, numerator_cols, date_col, scales, logger):
                 logger.error("Unable to calculate weekday correction")
             else:
                 params[i,:] = result
-
+        # the values multiplied by in calc_adjustment
+        # Sunday, the last day here, is a sum of the other days
+        effective_multipliers = np.exp(np.concatenate([-params[:,:6], params[:,:6].sum(axis=0, keepdims = True)]))
+
+        if effective_multipliers.max() == np.inf or effective_multipliers.min() == -np.inf:
+            logger.warning("largest weekday correction is infinite. Defaulting to no correction")
+            params = np.zeros((nums.shape[1], X.shape[1]))
```

Comment:
maybe make this replacement optional?  itd be nice if we could bubble up to the caller that this has happened, but im not sure how to do that nicely without throwing an exception (which defeats the purpose) or by changing the signature on the return (which might break some uses of this class)...  perhaps with a "pass by reference"-style argument that this method can modify?  thats kinda uglor but ¯\\\_(ツ)_/¯

Reply:
in this case, the values are literally impossible to use, since it's multiplying by infinity. I suppose it should probably just kill the whole process.

note that passing all zeros is interpreted as a failure by `doctor-visits`, see here: https://github.com/cmu-delphi/covidcast-indicators/blob/0c65bb468d9aa3a6c2f37d6f28a5490029b32354/doctor_visits/delphi_doctor_visits/update_sensor.py#L136-L138